packages/grafana-ui/src/components/Typeahead/TypeaheadInfo.tsx

```diff
@@ -41,15 +41,15 @@ interface Props {
 export const TypeaheadInfo: React.FC<Props> = ({ item, height }) => {
   const visible = item && !!item.documentation;
   const label = item ? item.label : '';
-  const documentation = item && item.documentation ? item.documentation : '';
+  const documentation = renderMarkdown(item?.documentation);
```

Comment:
Should probably sanitize the produced html with `textUtil.sanitize()`

Reply:
Done